subsys/bluetooth/controller/Kconfig

```diff
@@ -152,6 +152,7 @@ comment "BLE Controller configuration"
 config BT_CTLR_CRYPTO
 	bool "Enable crypto functions in Controller"
 	default y
+	select ENTROPY_GENERATOR
```

Comment:
Remove from line number 71 and 81. And move the RNG related Kconfig too here.

Reply:
What if the user deselects BT_CTLR_CRYPTO? LL_SPLIT and LL_LEGACY still has the dependency on ENTROPY_GENERATOR. Also the specific entropy generator would not work for all use-cases. I think it needs to remain like this. Possible change to `select ENTROPY_GENERATOR if soc_compatible_nrf`

Reply:
Ok. no change required, no harm in having the select in 71 and 81.